addons/mail/static/src/views/activity/activity_model.js

```diff
@@ -13,7 +13,7 @@ export class ActivityModel extends RelationalModel {
 
     async load(params = {}) {
         this.originalDomain = params.domain ? [...params.domain] : [];
-        params.domain = [...params.domain, ["activity_ids", "!=", false]];
+        params.domain = params.domain ? [...params.domain, ["activity_ids", "!=", false]] : [["activity_ids", "!=", false]];
```

Comment:
I think there's also an issue related to that bug, in `/documents/static/src/views/hooks.js`, line 367, we should do
```js
await env.model.load(component.props);
```

instead of 
```js
await env.model.load();
```

(That will fix the traceback, but both issue should be fixed imo)

Like they did in that PR: 768228690e163956622c2059ee0d27962872f6d2 (that introduced the bug, can you include the hash in your commit message for reference ?)

Side note, we can do something a little shorter here
```js
params.domain = [...(params.domain || []), ["activity_ids", "!=", false]];
```

Reply:
Oupsie :D

Reply:
Hello : ) @tde-banana-odoo 
I've updated the PRs from saas-16.3 with the optimization suggested by @std-odoo, and included the commit hash of the PR that introduced this bug in my commit message as well.

Regarding the aforementioned issue of domain not applied after replacing docs, agreeably, it can be resolved by sending component props. Since the fix was in enterprise, I've created this [PR](https://github.com/odoo/enterprise/pull/55716) targetting saas-16.2 for the same.

Now about the test case, I've recently discovered a lot of bugs and issues with the activity view and its document inspector in particular. Hence, I suggest that we create a task intended to introduce a new test file for the documents activity view and its inspector functionalities, since none of that is being tested at present, and it doesn't seem right to place the test case in some unrelated file.

Reply:
@khpa-odoo agree :) and we can try to centralize issues in one task / one PR instead of multiple small bugfixes that are sometimes forgotten :) feel free to create the task and proceed with it

Reply:
Sure, noted : )